bigtable/admin.go

```diff
@@ -119,6 +120,72 @@ func (ac *AdminClient) backupPath(cluster, backup string) string {
 	return fmt.Sprintf("projects/%s/instances/%s/clusters/%s/backups/%s", ac.project, ac.instance, cluster, backup)
 }
 
+// EncryptionInfo represents the encryption info of a table.
+type EncryptionInfo struct {
+	EncryptionStatus *Status
```

Comment:
Minor nit: there is a lot of repeating of "Encryption" here, can you simplify the naming?

Reply:
I understand this point of view. I mirrored the proto messages, but could consider removing some of the repetition. 

https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/table.proto#L219

Reply:
ended up removing repetition.